### DISCUSSION TITLE
How to drag groups in appflowy board

### DISCUSSION THREAD

**Discussion by quasar-pankaj** [AppFlowy-IO/AppFlowy#1185] [General help wanted]
Hello,

I have been playing around with the sample app. I am planning to use appflowy board in my app which requires similar features. The sample provided doesn't seem to drag and drop groups. I looked at the API docs but couldn't find anything relevant. Can anybody help?

Thanks in advance,
Regards,
Pankaj

**Comment by appflowy** [comment]
> Hello,
> 
> I have been playing around with the sample app. I am planning to use appflowy board in my app which requires similar features. The sample provided doesn't seem to drag and drop groups. I looked at the API docs but couldn't find anything relevant. Can anybody help?
> 
> Thanks in advance, Regards, Pankaj

Hi, @quasar-pankaj . The d`rag and drop groups` feature is a work in progress.  I will let you know after I finish it.

**Reply by quasar-pankaj** [reply]
No problem. Thanks. I found another problem - The horizontal scrolling seems not to work. I am using Linux-desktop if the helps.

**Comment by appflowy** [comment]
@quasar-pankaj I published the v0.0.8 version that supports drag and drop groups. You could have a try.
